### ISSUE TITLE
Set up our own blog

### ISSUE THREAD

**Issue by ericdill** [conda-forge/conda-forge.github.io#916]
There was some discussion a few meetings back about us setting up our own blog. We tossed a few ideas out like using Medium (which was generally opposed) and hosting some Jekyll blog on github with gh-pages (was more supported). As of right now it's been on @ocefpaf 's plate for a month or two but due to we're all too busy, he just hasn't had time to make any progress on it. This issue is officially taking it off of @ocefpaf 's plate so that someone else can pick it up and run with it. (Thanks for everything else that you do for conda-forge Filipe!) As of right now I'm not really sure what our requirements would even be for setting up a blog. I guess step 1 is having someone with blog content ready to go. That would likely force us to make a decision on something. I think @CJ-Wright had something ready to go at some point. Maybe that'll be the shove we need to finally get a blog going.

**Comment by CJ-Wright** [comment]
I made a blog repo here: https://github.com/conda-forge/blog and put up a PR for some news. I didn't populate it with anything though.

**Comment by ocefpaf** [comment]
> As of right now it's been on @ocefpaf 's plate for a month or two but due to we're all too busy, he just hasn't had time to make any progress on it. This issue is officially taking it off of @ocefpaf 's plate so that someone else can pick it up and run with it.

Yep. I won't be able to move this forward. BTW, I asked multiple times about platforms and what would be ideal and got little feedback or only "tools that we should not use," but no advice on what tool would be nice.

TL;DR whoever picks this up just do whatever you works and we'll adapt to it.

**Comment by Deathrow77** [comment]
@ocefpaf @CJ-Wright I would like to make a suggestion here. For a site that is not updated daily and has a simple structure, its better to choose a static hosting provider that is secure and fast such as GitHub Pages, GitLab, Dropbox, Amazon S3, etc and generate content with a static site generator such as Hugo, Jekyll, Harp, Hexo, Midlemman, etc.

Hugo sites can be hosted anywhere has integrated Disqus comment support, support for theming, customizable URL's, taxonomies, content sorting, dynamic menus, SEO friendly URL's, permalinks and redirects.

With a static site generator, you didn’t really lose anything of practical value. Instead, you have several advantages such as speed, security, version control, small footprint, more environment-friendly. 

Also, I would like to take up this issue.

**Comment by CJ-Wright** [comment]
@Deathrow77 Great! Thank you very much!

What can we do to help enable you?

**Comment by Deathrow77** [comment]
@CJ-Wright .. All I need is your permission to start working on this project

**Comment by Deathrow77** [comment]
Also, I would love to know more about what this blog is supposed to be used for? Are you guys planning on writing about your project milestones/improvements or is this supposed to be a wiki type of thing to know more about the project?

**Comment by jakirkham** [comment]
It’s probably random articles. Possibly about improvements, maybe advice, could be other postings as well

**Comment by ocefpaf** [comment]
@Deathrow77 please take a look at https://github.com/conda-forge/blog/pull/2 and, if possible, let's move this discussion there.
